application/helpers/SurveyRuntimeHelper.php

```diff
@@ -433,7 +433,7 @@ public function run($surveyid, $args)
 
             $aGroup['showgroupinfo'] = $showgroupinfo_;
             $aGroup['showdescription']  = (!$this->previewquestion && trim($gl['description']) != "" && $showgroupdesc_);
-            $aGroup['description']      = $gl['description'];
+            $aGroup['description']      = LimeExpressionManager::ProcessString($gl['description'], null, null, 3, 1);
```

Comment:
We process 2 times then ? 

Seems GROUPNAME have an issue, but process description is another issue (and i don't think we need it).

Reply:
https://github.com/LimeSurvey/LimeSurvey/blob/435badbd8f103673ece463a6c2bea8a4296ff25f/themes/survey/vanilla/views/subviews/survey/group_subviews/group_desc.twig#L25

Reply:
Yes, I agree.